chapters/samplers.txt

```diff
@@ -478,6 +478,51 @@ ifdef::VK_VERSION_1_2[<<features-samplerMirrorClampToEdge,samplerMirrorClampToEd
 endif::VK_VERSION_1_2,VK_KHR_sampler_mirror_clamp_to_edge[]
 --
 
+[open,refpage='VkCompareOp',desc='Comparison operator for depth, stencil, and sampler operations',type='enums']
+--
+_Comparison operators_ compare a _reference_ and a _target_ value, and
+return a true or false value depending on the comparison operator chosen.
+The supported operators are:
+
+include::{generated}/api/enums/VkCompareOp.txt[]
+
+  * ename:VK_COMPARE_OP_NEVER specifies that the test evaluates to false.
+  * ename:VK_COMPARE_OP_LESS specifies that the test evaluates
+    [eq]#_reference_ < _target_#.
+  * ename:VK_COMPARE_OP_EQUAL specifies that the test evaluates
+    [eq]#_reference_ = _target_#.
+  * ename:VK_COMPARE_OP_LESS_OR_EQUAL specifies that the test evaluates
+    [eq]#_reference_ {leq} _target_#.
+  * ename:VK_COMPARE_OP_GREATER specifies that the test evaluates
+    [eq]#_reference_ > _target_#.
+  * ename:VK_COMPARE_OP_NOT_EQUAL specifies that the test evaluates
+    [eq]#_reference_ {neq} _target_#.
+  * ename:VK_COMPARE_OP_GREATER_OR_EQUAL specifies that the test evaluates
+    [eq]#_reference_ {geq} _target_#.
+  * ename:VK_COMPARE_OP_ALWAYS specifies that the test evaluates to true.
+
+Comparison operators are used for:
```

Comment:
With such a nice navigation map, it is a question if not move it to Common Object Types chapter.

Reply:
The style is to define a common enum in the first place it is used, though I could see a case for this. Will keep it in mind but not as part of this PR.